Doc/whatsnew/3.11.rst

```diff
@@ -1677,6 +1682,9 @@ Build Changes
 * The :mod:`tkinter` package now requires Tcl/Tk version 8.5.12 or newer.
   (Contributed by Serhiy Storchaka in :issue:`46996`.)
 
+* :mod:`tkinter` now requires ``pkg-config`` command to detect development
+  settings for headers and libraries. 
```

Comment:
We should incorporate this into the bullet point in lines 1641-1644 (line numbers in this PR).